### PR TITLE
Fix/double requests

### PR DESCRIPTION
**Pull request by octavioamu**
##### Description

remove query on view load of grant explorer, instead just use the js request for the api.

### CODE REVIEW

File: app/dashboard/tasks.py [hunk 1]
```diff
@@ -294,6 +294,7 @@ def increment_view_count(self, pks, content_type, user_id, view_type, retry: boo
     :param view_type:
     :return:
     """
+    print(pks, content_type, user_id, view_type)
```
Comment (gdixon) [inline app/dashboard/tasks.py:297]: Consider: dropping this print

File: app/grants/views.py [hunk 2]
```diff
@@ -595,6 +595,11 @@ def get_grants(request):
             del grant_json['weighted_risk_score']
         grants_array.append(grant_json)
 
+    pks = list([grant.pk for grant in grants])
+    if len(pks):
+        print(request.user)
```
Comment (gdixon) [inline app/grants/views.py:600]: And this one

File: app/grants/views.py [hunk 3]
```diff
@@ -867,6 +872,7 @@ def grants_landing(request):
 
 def grants_by_grant_type(request, grant_type):
     """Handle grants explorer."""
+    print(request)
```
Comment (gdixon) [inline app/grants/views.py:875]: And this one

**Review by gdixon — APPROVED** [review]
Looking awesome @octavioamu!! 🚀

**Comment by thelostone-mc** [comment]
Could we remove the print and move this to master as opposed to stable ? 
codewise LGTM

**Comment by octavioamu** [comment]
Allll done

**Pull request merged**